Library/Homebrew/extend/kernel.rb

```diff
@@ -196,9 +225,10 @@ def disk_usage_readable(size_in_bytes)
     end
   end
 
+  sig { params(number: Integer).returns(String) }
   def number_readable(number)
     numstr = number.to_i.to_s
```

Comment:
`i.to_i` is a little silly, we know `i` is an int based on the `step` size. We could also fix this with `T.cast(i, Integer)` instead, though that's a bit harsher on the eyes.